### DISCUSSION TITLE
Rebuild model

### DISCUSSION THREAD

**Discussion by beppo-ivel** [appcelerator/titanium_mobile#12831] [Q&A] [answered]
In the beginning of developing an idea, I renew my model sometimes. Till now there for I have to do the following steps:

>     create a migration file, where migration.up function drops the table aka model
>     rebuild the app
>     change my model file
>     delete migration file
>     rebuild app

What is a proper solution to rebuild my model without managing migration files?

**Comment by m1ga** [comment, marked as answer]
is it an app that is in the store already or are you still developing it? If it is just a dev app (and the model data is generated or doesn't matter) you can just modify the model file and remove the app (or delete the app data on android) and reinstall/rerun the app. If the app is live already you'll need to keep the migration files for older databases/apps. Otherwise you'll get some errors there when people update your app
